### PR TITLE
feat: add "Current Location" as a selectable option.

### PR DESCRIPTION
**Pull request by taghizadlaura**
# Add "Current Location" as a selectable option.

## Description
This PR introduces the UI for the "Use Current Location" feature on the Create Alert screen. Users can now see an option to set their location automatically using the current location, though the location logic is not yet implemented. This closes issue #181 (and so the issue #151).

## Changes
- Added a "Current Location" DropdownMenuItem to the Create Alert screen, allowing users to trigger the location setting.

#### Modified
- `CreateAlertScreen.kt`: Contains the "Current Location" option and a specific icon.

## Testing
- Since we didn't specifically test the dropdown items in any other field, I assumed that all dropdown-related tests would be handled later as part of the broader testing strategy.

### CODE REVIEW

File: app/src/main/java/com/android/periodpals/ui/alert/CreateAlert.kt [hunk 1]
```diff
@@ -188,6 +191,16 @@ fun CreateAlertScreen(
               onDismissRequest = { showDropdown = false },
               modifier = Modifier.wrapContentSize(),
           ) {
+            DropdownMenuItem(
+                text = { Text("Current Location") },
```
Comment (charliemangano) [inline app/src/main/java/com/android/periodpals/ui/alert/CreateAlert.kt:195]: Can you extract this to a constant?

File: app/src/main/java/com/android/periodpals/ui/alert/CreateAlert.kt [hunk 2]
```diff
@@ -188,6 +191,16 @@ fun CreateAlertScreen(
...
+                onClick = {
+                  // Logic for fetching and setting current location will be implemented later
```
Comment (charliemangano) [inline app/src/main/java/com/android/periodpals/ui/alert/CreateAlert.kt:197]: Nitpick but can you mark this comment as a TODO, it makes it easier to see what's left to do when planning tasks :)

**Review by charliemangano — APPROVED** [review]
LGTM, good job! Take a look at the comments for small nitpicks

**Review by francelu — APPROVED** [review]
## Summary
**LGTM**  
Minor suggestions/nitpicks: echoing what @charliemangano pointed out.  

Additionally, I noticed that this menu is a drop-down but it placed **up** the location field. It might be worth placing it below the location field for better UX, though I recognize this change may be out of scope for this PR. However, since you're working on location-related elements, it's something to consider in the future.


### Important (which includes)
- **Code Quality**: Good :)
- **Functionality**: Improved by this PR with the "Set current location"
- **Performance**: -
- **Security**: -

### Testing
Ensure adequate testing for all new changes and edge cases.

### Documentation
Verify if any updates to the documentation are necessary.

### Suggestions
Consider UX adjustments, particularly related to menu placement, in a future iteration.  
Also, it would be nice to have a consistent layout. (It is already good, but) Check the wiki page if you have time :)

### Steps before PR merge
- Explicitly address the suggestions and nitpicks raised.

**Comment by taghizadlaura** [comment]
@francelu, you mentioned, 'I noticed that this menu is a drop-down but it’s placed above the location field. It might be worth placing it below the location field for better UX, though I recognize this change may be out of scope for this PR.' Do you want me to move it below all the options? Or should we wait for the Figma to be updated and discuss the UI later with the group?

**Comment by francelu** [comment]
> @francelu, you mentioned, 'I noticed that this menu is a drop-down but it’s placed above the location field. It might be worth placing it below the location field for better UX, though I recognize this change may be out of scope for this PR.' Do you want me to move it below all the options? Or should we wait for the Figma to be updated and discuss the UI later with the group?

@taghizadlaura  
I think we will change the layout later on, when we will have updated the Figma. Same for padding, typography and colours.

**Pull request merged**